README.md

```diff
@@ -5,8 +5,15 @@ This project replaces the legacy "VIVO Installer". It is used to create a "third
 ## Usage
 
 1. Clone this repository
-1. Add any overlay files to their expected location with the `src` directory tree
-1. Build the custom webapp
+2. Add any overlay files to their expected location with the `src` directory tree
+3. Add any overlay files to their expected location with the `home` directory tree
```

Comment:
These instructions are very helpful.
It may be good to include in this pull-request an empty directory tree:
```
home/src/main/resources/config/
home/src/main/resources/rdf/
```
..so it is even more clear where developers should put their overrides.

Reply:
Would have to include .gitkeep or such files. Will that be ok? Can exclude them from the resource copy overlay.

Reply:
Sounds good.